src/pages/community-partners.astro

```diff
@@ -0,0 +1,91 @@
+---
+import Layout from '@layouts/Default.astro'
+
+const partners = [
+  {
+    'name': 'TheAlgorithms',
+    'image': 'https://github.com/TheAlgorithms.png',
+    'link': 'https://thealgorithms.com',
+    'description': 'Open Source resource for learning Data Structures & Algorithms and their implementation in any Programming Language'
+  },
+  {
+    'name': 'GitHub',
+    'image': 'https://github.com/GitHub.png',
+    'link': 'https://github.com',
+    'description': 'GitHub is one of the world’s largest community of developers. It’s an intricate platform that fosters collaboration and communication between developers.'
+  },
+  {
+    'name': 'Web3Community',
+    'image': 'https://github.com/Web3Community.png',
+    'link': 'https://web3community.github.io',
+    'description': 'An Open Source Community that focuses on decentralized applications, web 3, and blockchain technologies 🚀'
+  }
+]
+---
+
+<Layout>
+  <ul
+    role="list"
+    class="
+      mt-20
+      p-20
+      flex
+      flex-wrap
+      min-w-min
+      max-w-max
+      items-center
+      justify-center
+    "
+  >
+    {partners.map(partner => (
+    <li
+      class="
+        m-8
+        col-span-1
+        flex flex-col
+        text-center
+        bg-white
```

Comment:
Maybe we can change it to black when in dark mode and keep it white in light mode.
I think I do know how to change colors when the theme changes. 🙂

I used Dev Tools on the web, modified it and this is how it looks.

![image](https://user-images.githubusercontent.com/51391473/143086943-ea0d0a47-6890-4478-97fe-996c8ab0666f.png)

Reply:
I think keeping the opposites would be better in terms of UX